cmd/create.go

```diff
@@ -106,16 +107,18 @@ to quickly create a Cobra application.`,
 		informUser("ArgoCD Ready")
 		progressPrinter.IncrementTracker("step-argo", 1)
 
-		var kPortForwardArgocdOutb, kPortForwardArgocdErrb bytes.Buffer
-		kPortForwardArgocd := exec.Command(config.KubectlClientPath, "--kubeconfig", config.KubeConfigPath, "-n", "argocd", "port-forward", "svc/argocd-server", "8080:80")
-		kPortForwardArgocd.Stdout = &kPortForwardArgocdOutb
-		kPortForwardArgocd.Stderr = &kPortForwardArgocdErrb
-		err = kPortForwardArgocd.Start()
-		defer kPortForwardArgocd.Process.Signal(syscall.SIGTERM)
-		if err != nil {
-			log.Printf("Commad Execution STDOUT: %s", kPortForwardArgocdOutb.String())
-			log.Printf("Commad Execution STDERR: %s", kPortForwardArgocdErrb.String())
-			log.Panicf("error: failed to port-forward to argocd in main thread %s", err)
+		if !dryRun {
+			var kPortForwardArgocdOutb, kPortForwardArgocdErrb bytes.Buffer
+			kPortForwardArgocd = exec.Command(config.KubectlClientPath, "--kubeconfig", config.KubeConfigPath, "-n", "argocd", "port-forward", "svc/argocd-server", "8080:80")
+			kPortForwardArgocd.Stdout = &kPortForwardArgocdOutb
```

Comment:
We need to double check if we are not breaking `main`here.. seems to be some old code getting a ride on create.